sklearn/cluster/birch.py

```diff
@@ -74,7 +74,7 @@ def _split_node(node, threshold, branching_factor):
 
     farthest_idx = np.unravel_index(
         dist.argmax(), (n_clusters, n_clusters))
-    node1_dist, node2_dist = dist[[farthest_idx]]
+    node1_dist, node2_dist = dist[tuple([farthest_idx])]
```

Comment:
I'm still confused about whether this is doing the same as it was before

Reply:
The list was wrapping `farthest_idx` (a tuple) inside a list, now it's a tuple inside a tuple - I'm a bit confused too even though it's passing.